_tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot.md

```diff
@@ -16,11 +16,19 @@ The searchable snapshot feature incorporates techniques like caching frequently
 
 ## Configuring a node to use searchable snapshots
 
-To configure the searchable snapshots feature, create a node in your opensearch.yml file and define the node role as `search`:
+To configure the searchable snapshots feature, create a node in your opensearch.yml file and define the node role as `search`. Optionally, also configure the `cache.size` property for the node.
+
+A `search` node reserves storage for the cache for performing searchable snapshot queries. In case of a dedicated search node (where the node exclusively has the `search` role), this value defaults to a fixed percentage of available storage. In other cases, the value needs to be configured by the user.
```

Comment:
```suggestion
A `search` node reserves storage in the cache for performing searchable snapshot queries. In the case of a dedicated search node where the node exclusively has the `search` role, this value defaults to a fixed percentage of available storage. In other cases, the value needs to be configured by the user using the `node.search.cache.size` setting.
```

Reply:
Thanks @Naarcha-AWS! I want to convey "... the node reserves storage space on disk for the cache to perform searchable snapshot queries" here.

Reply:
@kotwanikunal: Is the storage contained in the cache? I'm not sure what the distinction of switching it to "for the cache" is supposed to convey technically.

Reply:
The cache utilizes the storage on the node, so it reserves/marks a part of storage as utilized

Reply:
The switch is to convey `node reserves storage for the cache` instead of `node reserves storage in cache` -
cache consumes storage and not the other way around.